src/google-maps/google-map/google-map.ts

```diff
@@ -83,6 +84,12 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
   /** Width of the map. Set this to `null` if you'd like to control the width through CSS. */
   @Input() width: string | number | null = DEFAULT_WIDTH;
 
+  /**
+   * The Map ID of the map. This parameter cannot be set or changed after a map is instantiated.
+   * See: https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions.mapId
+   */
+  @Input() mapId: string | undefined;
```

Comment:
Why is this input being added? I don't see it being used in the advanced marker.

Reply:
You need to specify the mapId when using advanced markers. Please refer to my screenshot below.

<img width="1498" alt="Screenshot 2024-02-07 at 12 52 36" src="https://github.com/angular/components/assets/45272165/9481adc9-cded-42c5-822c-dc6fe99f4cdc">